### PR TITLE
Background color changer project PR from branch 

### PR DESCRIPTION
**Pull request by BajraYeJoon**

### CODE REVIEW

**Review by Harshil-Jani — COMMENTED** [review]
Looks Good to me :100:

**Comment by Harshil-Jani** [comment]
We will merge it from tomorrow.. @BajraYeJoon Let Hacktoberfest begun ! :tada:

File: Background Color Changer/index.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
```
Comment (soumik2012) [inline Background Color Changer/index.html:1]: - Write a folder name  `Background_Color_Changer` not this `Background Color Changer`

**Review by soumik2012 — CHANGES_REQUESTED** [review]
Add Readme.md File of Background Color it should be contained 
- Name of Project
- Demo of Project in .gif how your project is look 
- Description of project 
-  Requirements or Setup instructions
-  Name of Author(s)  
- [Look Here](https://github.com/Tenet-Coding/Hacktoberfest-Projects/blob/master/ConwaysGameOfLife/README.md)

**Comment by BajraYeJoon** [comment]
All Changes requested are commited

**Comment by Harshil-Jani** [comment]
@BajraYeJoon Just add the .gif for your working project int README.md and then we are good to go :star2:

**Comment by BajraYeJoon** [comment]
Readme.md is updated with requested changes

**Review by soumik2012 — APPROVED** [review]
LGTM 💯

**Review by Harshil-Jani — COMMENTED** [review]
We are good to go :tada:

**Review by Harshil-Jani — APPROVED** [review]

**Comment by soumik2012** [comment]
@BajraYeJoon It will be showing now 
Thank you for contributing to @tenet-coding 🥳 🍾

**Pull request merged**